.github/workflows/reusable-docker-publish.yml

```
@@ -117,6 +117,7 @@ jobs:
           build-args: |
             CKAN_REF=${{ env.CKAN_REF }}
             ENV=base
+            ${{ inputs.ckan-major-version == '2.11' && 'PYTHON_VERSION=${{ env.PY_VERSION_VALUE }}' || '' }}
```

Checking the version won't scale nicely when new versions come up. What about checking if `env.PY_VERSION_VALUE` is defined? Also we are double wrapping the env var so I'm not sure that would work. What about something like:

```suggestion
            ${{ env.PY_VERSION_VALUE && 'PYTHON_VERSION=${{ env.PY_VERSION_VALUE }}' || '' }}
```